docs/user_interface/how_to_install_ui_locally.rst

```diff
@@ -0,0 +1,53 @@
+.. _how-to-install-ui-locally-page:
+
+=========================
+How To Install UI Locally
+=========================
+
+These installation instructions are targeted towards developers and those who wish to deploy the desktop application.
+
+
+**Prerequisites**
+
+- Miniconda
+- npm
+
+
+Installing the UI
+-----------------
+
+1. Fork the repo on GitHub (your copy of the main repo)
+
+2. Clone your fork locally, creating a workspace (replacing ``<githubid>`` with your github user id)::
+
+    git clone https://github.com/<githubid>/pareto-ui
+
+3. Create a dedicated Conda environment for development work::
+
+    conda env create --file environment.yml
```

Comment:
Where does the `environment.yml` file used in this command come from? Also, does this command need to name the environment `pareto-ui-env` as it is referred to in the next step?

Reply:
@tarnold17 The `environment.yml` file is part of the UI repo. It specifies the name of the environment (`pareto-ui-env`), the python and node versions, and points to the requirements.txt file to install the required python packages. While it's not entirely necessary to use this file to create the environment, using it limits the possibility of version incompatibilities.

Reply:
@MichaelPesce got it. Can you update the instructions here so it's clear that you are referring to the `environment.yml` file in the top level of the pareto-ui repo? Also, I see now that the name `pareto-ui-env` is part of the `environment.yml` file. That detail might also be worth mentioning - perhaps something like "The following command creates a dedicated Conda environment for development work called pareto-ui-env:".

Reply:
how about: 
"Navigate into the new `pareto-ui` directory, and run the following command to create a dedicated Conda environment for development work called `pareto-ui-env`:"

Reply:
@MichaelPesce that sounds great.